sub_cal/sub_cal/src/main/java/App.java

```diff
@@ -0,0 +1,64 @@
+
+import io.Input;
+import io.Output;
+import model.Calculator;
+import model.HistoryEntity;
+import option.Option;
+
+import java.io.IOException;
+import java.util.NoSuchElementException;
+
+public class App implements Runnable{
+    private final Input input;
+    private final Output output;
+    private final HistoryEntity historyEntity;
+    private final Calculator calculator;
+
+    public App(Console console, HistoryEntity historyEntity, Calculator calculator) {
+        this.input = console;
+        this.output = console;
+        this.historyEntity = historyEntity;
+        this.calculator = calculator;
+    }
+
+    @Override
+    public void run(){
+
+        while (true) {
+
+            output.printMenuList();
+            Option select = null;
+            try{
+             select = input.selectOption();
+            } catch (NoSuchElementException e1){
+                continue;
+            } catch (IOException e2){
+                continue;
+            }
```

Comment:
```suggestion
            } catch (NoSuchElementException | IOException e1){
                continue;
            }
```
같은 일을 한다면 이렇게 해주면 좋을 것 같아요

Reply:
넵 수정하였습니다